INSTALL.md

```diff
@@ -1,24 +1,31 @@
 ## Installation
 
 ### Requirements
+
 - Linux, CUDA >= 11.7, Python >= 3.8, PyTorch >= 2.0.0 (our setup below is based on CUDA 11.8, Python 3.10, PyTorch 2.0.1; more recent versions should work too, but no guarantees)
 - Conda (anaconda / miniconda work well)
+- visual studio visual studio (2022/2019/2017) build tools
 
 ### Setup
 
 #### 1. Create conda environment
+
 ```bash
-conda create --name ssvp_slt python=3.10
+conda create --name ssvp_slt python=3.10 cmake
 conda activate ssvp_slt
 conda install pytorch==2.0.1 torchvision==0.15.2 pytorch-cuda=11.8 -c pytorch -c nvidia
 conda install av -c conda-forge
+python -m pip install torch==2.2.0 torchvision==0.17.0 torchaudio==2.2.0 --index-url https://download.pytorch.org/whl/cu118
```

Comment:
You can remove L16 since `torch` and `torchvision` are also defined there. Let's just specify one version of each library.